patternlab/styleguide/source/_patterns/03-organisms/feedback/mass-feedback-contact-requested.json

```diff
@@ -0,0 +1,85 @@
+{
+  "formAction": "#",
+  "title": "Did you find what you were looking for on this webpage?",
+  "org": "the Registry of Motor Vehicles",
+  "queryAffirmative": "Do you have anything else to tell us?",
+  "queryNegative": "Please tell us what you were looking for.",
+  "queryDescription": "We will use this information to improve the site.",
+  "inputGroup": {
+    "required": true,
+    "inline": true,
+    "title": "",
+    "type": "radio",
+    "items": [
+      {
+        "name": "foundIndicator",
+        "value": "yes",
+        "id": "field1",
+        "label": "Yes",
+        "checked": false
+      },
+      {
+        "name": "foundIndicator",
+        "value": "no",
+        "id": "field2",
+        "label": "No",
+        "checked": false
+      }
+    ]
+  },
+  "textarea": {
+    "labelText": "Text Area",
+    "hiddenLabel": true,
+    "required": false,
+    "maxlength": 1000,
+    "id": "textarea",
+    "name": "textarea"
+  },
+  "helperText": {
+    "inputId": "GUID2738572489",
+    "message": "Do not include sensetive information, such as Social Security or bank account numbers."
+  },
+  "Response": "Would you like a response from",
+  "responseRequired": {
+    "responseDescription": "Responses may take 1 to 3 days",
+    "inputGroup": {
+      "required": true,
+      "inline": true,
+      "title": "",
+      "type": "radio",
+      "items": [
+        {
+          "name": "contactGroups",
+          "value": "yes",
+          "id": "field3",
+          "label": "Yes",
+          "checked": false
+        },
+        {
+          "name": "contactGroups",
+          "value": "no",
+          "id": "field4",
+          "label": "No",
+          "checked": false
+        }
+      ]
+    }
+  },
+  "currentPage": {
```

Comment:
I think this should use the format that I pushed to the mass-feedback-form.json: see "hiddenElements".